cypress/e2e/scrape.server.cy.ts

```diff
@@ -29,6 +29,34 @@ describe("getLatestSnapshot", () => {
       });
   });
 
+  it("should ensure that the return timestamp from wayback machine is not too far into the future", () => {
```

Comment:
nit: we can just include this  in the first test.  Get the one timeStampDate and then make two assertions (expect) against it.

Reply:
Yes we can. Fixed this.